templates/docs/patterns/search-and-filter.md

```diff
@@ -0,0 +1,95 @@
+
+Please use toggle the `aria-hidden` attribute on the `.p-search-and-filter__panel` element when you want to reveal the filter panel. When a user makes a selection you can append a [chip element](/docs/patterns/chip) to the `.search-and-filter__search-container` element.
+
+### Overflowing chips
+
+When the selection reaches the end of the container the chips wrap and should display the number of wrapping chips in the selected-count element. The element becomes a hit target to expand the selected chips section.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/search-and-filter/chip-overflow/" class="js-example">
+View example of the search and filter chip overflow pattern
+</a></div>
+
+#### Functionality
+
+When a click event is triggered on the selected-count element you need need to toggle three states.
+
+- Switch `.p-search-and-filter__box` data-overflow attribute from false to true
+- Switch `.p-search-and-filter__panel` aria-hidden attribute from true to false
+- Switch `.p-search-and-filter__search-container` aria-expanded attribute from false to true
+
+<div class="p-notification--information">
+  <p class="p-notification__response" role="status">You will have to handle the overflowing chip count yourself. Unfortunately that is out of scope for this example but this can be handled for you by the <a href="#react">React component</a>.</p>
+</div>
+
+### Search prompt
+
+When you type in the search input the behaviour should be that the available selections are filtered to match the input. Also, a raw string input should be made available.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/search-and-filter/with-search-prompt/" class="js-example">
+View example of the search and filter with search prompt pattern
+</a></div>
+
+#### Functionality
+
+<div class="p-notification--information">
+  <p class="p-notification__response" role="status">You will have to handle filtering the available chips yourself by removing or hiding chips that do not match. Unfortunately that is out of scope for this example but this can be handled for you by the <a href="#react">React component</a>.</p>
+</div>
+
+## Import
+
+To import just this component into your project, copy the snippet below and include it in your main Sass file.
+
+```scss
+@import '../base';
+@include vf-base;
+
+@import '../patterns_chip';
+@import '../patterns_search-box';
+@import '../patterns_search-and-filter';
+@import '../utilities_off-screen';
+@include vf-p-chip;
+@include vf-p-search-box;
+@include vf-p-search-and-filter;
+@include vf-u-off-screen;
+```
+
+For more information see [Customising Vanilla](/docs/customising-vanilla/) in your projects, which includes overrides and importing instructions.
+
+## React
+
+For use of the React component you can do that by installing our react-component page and importing.
+
+[Playbook example](https://canonical-web-and-design.github.io/react-components/?path=/docs/search-and-filter--default-story)
```

Comment:
Did you mean "Storybook" instead of playbook? Also, I'm not sure if it's useful to call it by the name of the documentation tool. Maybe something like "See the React documentation for SearchAndFilter component"?

```suggestion
[See the documentation for our React `SearchAndFilter` component](https://canonical-web-and-design.github.io/react-components/?path=/docs/search-and-filter--default-story)
```